spec/features/copying_files_spec.rb

```diff
@@ -19,7 +19,11 @@
     copy_attributes = copy.file.attributes
     copy_attributes.reject! { |k,v| k == :last_modified }
 
+    copy_acl_grants = copy.file.file.acl.grants
+    original_acl_grants = original.file.file.acl.grants
+
     expect(copy_attributes).to eq(original_attributes)
+    expect(copy_acl_grants).to eq(original_acl_grants)
```

Comment:
Did this happen to fail before the change was made? It looks like it could pass coincidentally.